pom.xml

```diff
@@ -67,14 +69,31 @@
         <dependency>
             <groupId>org.jvnet.hudson.plugins</groupId>
             <artifactId>analysis-core</artifactId>
-            <version>1.71</version>
-            <type>hpi</type>
+            <version>1.73-beta-SNAPSHOT</version><!-- TODO: change to 1.73 before release -->
```

Comment:
Thanks @amuniz ! Here's the test steps that I followed:

1. Created a simple job, using the XML files in src/test/resources
2. Tested that it was working correctly, and the graph was generated OK as well
3. Checked out pr/5 branch locally
4. Executed the job again
5. Everything worked OK, except the graph, that was empty
6. Updated to 1.73 the analysis-core. Same error.
7. Updated to 1.74, and now it worked!

Merging your pull request, but will bump analysis-core to 1.74.

Thanks and sorry for the delay to review and merge it.

Cheers,
Bruno